### ISSUE TITLE
Game breaking dupe bug

### ISSUE THREAD

**Issue by Shmeeb** [Xwaffle1/UniversalMarket#16]
when a player puts an item up for sale on the market, and another player buys it while the seller is on the item removal page, the seller can remove it after the person buys it, therefore duping the item

**Comment by Lorrainrm** [comment]
Confirming this

**Comment by Shmeeb** [comment]
@Xwaffle1

**Comment by RawrMaoster** [comment]
I can confirm this i just literally just tested this and im forced to disable the commands.

What i did to test this.

1) I put items in a satchel and put it up for sale for 1,000,000.
2) I gave my Operator 1,000,000 to buy it but only when i say to.
3) I clicked into the satchel where it gives you the options to buy and remove.
4) Leave open that window; i sent my operator a message in discord to buy the satchel and as soon as i seen + 1000000 on my screen i removed the item from the market and we both had the duped satchel with duplicated items.

I really hope this dupe gets fixed real soon or ill be forced to remove it :(
